RELEASES.md

```diff
@@ -22,172 +26,232 @@ Below are the versioning schemes used for publishing debs:
- 16. Once all SRU bugs are tagged as `verification*-done`, all SRU-bugs should be listed as green in [the pending_sru page](https://people.canonical.com/~ubuntu-archive/pending-sru.html). It is now time to ping the [current SRU vanguard](https://wiki.ubuntu.com/StableReleaseUpdates#Publishing) for acceptance of ubuntu-advantage-tools into -updates
+## Process
 
-## Ubuntu PRO Release Process
 
-Below is the procedure used to release ubuntu-advantage-tools to Ubuntu PRO images:
+### Background
 
- 1. [Open Daily PPA copy-package operation](https://code.launchpad.net/~ua-client/+archive/ubuntu/daily/+copy-packages)
- 2. Check Trusty, Xenial, Bionic package
- 3. Select Destination PPA: UA Client Premium [~ua-client/ubuntu/staging]
- 4. Select Destination series: The same series
- 5. Copy options: "Copy existing binaries"
- 6. Click Copy packages
- 7. Notify Pro Image creators about expected Premium PPA version (patviafore/powersj)
+The release process for ubuntu-advantage-tools has three overarching steps/goals.
 
+1. Release to our team infrastructure. This includes Github and the `ua-client` PPAs.
+2. Release to the latest ubuntu devel release.
+3. Release to the supported ubuntu past releases via [SRU](https://wiki.ubuntu.com/StableReleaseUpdates) using the [ubuntu-advantage-tools specific SRU process](https://wiki.ubuntu.com/UbuntuAdvantageToolsUpdates).
 
-## Release to PPA
+Generally speaking, these steps happen in order, but there is some overlap. Also we may backtrack if issues are found part way through the process.
 
-We manually upload the packages to our staging/stable PPAs. If you want to cut a new release and
-upload to one of these PPAs, follow these steps:
+An average release should take somewhere between 10 and 14 calendar days if things go smoothly, starting at the decision to release and ending at the new version being available in all supported ubuntu releases. Note that it is not 2 weeks of full time work. Most of the time is spent waiting for review or sitting in proposed.
+
+### Prerequisites
+
+If this is your first time releasing ubuntu-advantage-tools, you'll need to do the following before getting started:
+
+* Add the team helper scripts to your PATH: [uss-tableflip](https://github.com/canonical/uss-tableflip).
+* If you don't yet have a gpg key set up, follow the instructions
+  [here](https://help.launchpad.net/YourAccount/ImportingYourPGPKey) to create a key,
+  publish it to `hkp://keyserver.ubuntu.com`, and import it into Launchpad.
+* Before you run `sbuild-it` for the first time, you'll need to set up a chroot for each Ubuntu release.
+  Follow [these instructions](https://gist.github.com/smoser/14df5f0cd621e10d2282d7c90345e322#new-sbuild-creation)
+  to create a chroot for each supported release.
+* You must have launchpad already properly configured in your system in order to upload packages to the PPAs.
```

Comment:
I think we can this link here for reference:
https://help.launchpad.net/Packaging/PPA/Uploading

Reply:
Good idea!